conf/messages.en

```diff
@@ -643,10 +644,11 @@ fileReady.p3.linkText = request another file
 downloadRequestSuccess.title = We''ve received your request for a file of your TGP data
 downloadRequestSuccess.h1 = We''ve received your request for a file of your TGP data
 downloadRequestSuccess.h2 = What happens next
-downloadRequestSuccess.p1 = We''ll send an email to {0} to let you know when the file is ready.
-downloadRequestSuccess.p2 = You will have a notification when you log in to your TGP with a link to the file download page. The file is in a comma separated value format.
-downloadRequestSuccess.p3 = This file will be available for 30 days from when we let you know it is ready.
-downloadRequestSuccess.p3.1 = After 30 days, you''ll need to request a new file.
+downloadRequestSuccess.p1.part1 = You can check the progress of your file on the
+downloadRequestSuccess.p1.linkText = download TGP data page.
```

Comment:
can we have this as {0} within part 1 because it will help with welsh translating